IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcTransportElement/DocEntity.xml

```diff
@@ -1,29 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think that the documentation could specify that this is a " a generalization of all **fixed** transport related objects".